### PR TITLE
[1LP][RFR] Fixed failing test cases in 5.10z for rhv provider

### PR DESCRIPTION
**Pull request by ganeshhubale**
This PR includes following changes:
- Two test cases: ```test_guest_os_shutdown``` and ```test_guest_os_reset``` are failing for ```rhv``` provider because of the BZs
 raised for both of these tests.
- These test cases were skipping for 5.10z before but As I see now blockers marker were removed accidentaly in any of the old PRs.
- Hence adding the blockes mark with related BZs and skipping these test cases for 5.10z
- These test cases were failing in 5.10 with below errors:
```E           SoftAssertionError: 
E           Last Boot Time value has not been refreshed (cfme/tests/infrastructure/test_vm_power_control.py:542)

cfme/fixtures/soft_assert.py:64: SoftAssertionError
SoftAssertionError

Last Boot Time value has not been refreshed (cfme/tests/infrastructure/test_vm_power_control.py:542)
===================================================================

>           raise TimedOutError("Could not do {} at {}:{} in time".format(message, filename, line_no))
E           TimedOutError: Could not do function _looking_for_state_change() at /var/ci/workspace/downstream-510z-tests-master/integration_tests/cfme/common/vm.py:742 in time

/var/ci/cfme_venv/lib/python2.7/site-packages/wait_for/__init__.py:180: TimedOutError
TimedOutError
Could not do function _looking_for_state_change() at /var/ci/workspace/downstream-510z-tests-master/integration_tests/cfme/common/vm.py:742 in time
```
- Fixed test case: ```test_no_template_power_control``` and updated with new behaviour in cfme

{{ pytest: cfme/tests/infrastructure/test_vm_power_control.py -k 'test_no_template_power_control or test_guest_os_shutdown or test_guest_os_reset'  -vv --use-provider=complete --long-running }}

### CODE REVIEW

File: cfme/tests/infrastructure/test_vm_power_control.py [hunk 1]
```diff
@@ -306,7 +306,6 @@ def test_power_on(self, appliance, testing_vm, ensure_vm_stopped, soft_assert):
         soft_assert(testing_vm.mgmt.is_running, "vm not running")
 
     @pytest.mark.rhv3
-    @pytest.mark.meta(automates=[BZ(1174858)])
```
Comment (lina-nikiforova) [inline cfme/tests/infrastructure/test_vm_power_control.py]: Why did you remove that?

File: cfme/tests/infrastructure/test_vm_power_control.py [hunk 2]
```diff
@@ -414,44 +411,41 @@ def test_no_template_power_control(provider, soft_assert):
...
+        if action == 'Power On':
+            action = 'Start'
+        elif action == 'Power Off':
+            action = 'Stop'
+        view.flash.assert_message('{} action does not apply to selected items'.format(action))
+        view.flash.dismiss()
 
     # Ensure there isn't a power button on the details page
     entity.click()
-    if provider.appliance.version < '5.10':
```
Comment (lina-nikiforova) [inline cfme/tests/infrastructure/test_vm_power_control.py]: :tada:

File: cfme/tests/infrastructure/test_vm_power_control.py [hunk 3]
```diff
@@ -522,6 +516,15 @@ def test_vm_power_options_from_off(provider, soft_assert, testing_vm, ensure_vm_
 
 
 @pytest.mark.provider([VMwareProvider, RHEVMProvider], override=True, scope='function')
+@pytest.mark.meta(
+    blockers=[
+        BZ(
+            1571830,
```
Comment (lina-nikiforova) [inline cfme/tests/infrastructure/test_vm_power_control.py]: Can you add this BZ in the docblock as well?

File: cfme/tests/infrastructure/test_vm_power_control.py [hunk 4]
```diff
@@ -552,6 +555,15 @@ def test_guest_os_reset(appliance, testing_vm_tools, ensure_vm_running, soft_ass
 
 
 @pytest.mark.provider([VMwareProvider, RHEVMProvider], override=True)
+@pytest.mark.meta(
+    blockers=[
+        BZ(
+            1571895,
```
Comment (lina-nikiforova) [inline cfme/tests/infrastructure/test_vm_power_control.py]: same as above

**Comment by lina-nikiforova** [comment]
@ganeshhubale LGTM, please address the comments and let me know

File: cfme/tests/infrastructure/test_vm_power_control.py [hunk 5]
```diff
@@ -376,29 +379,24 @@ def test_start_from_suspend(self, appliance, testing_vm, ensure_vm_suspended, so
...
+            1. An infra provider that has some templates.
+        testSteps:
+            1. Open the view of all templates of the provider
+            2. Verify the Power toolbar button is not visible
+            3. Select some template using the checkbox
+            4. Verify the Power toolbar button is not visible
+            5. Click on some template to get into the details page
+            6. Verify the Power toolbar button is not visible
+
+    Bugzillas:
```
Comment (izapolsk) [inline cfme/tests/infrastructure/test_vm_power_control.py]: Bugzilla
